### PR TITLE
[NFC][SYCL] Fix comment.

### PR DESCRIPTION
**Pull request by s-kanaev**

### CODE REVIEW

File: sycl/source/detail/scheduler/commands.hpp [hunk 1]
```diff
@@ -493,10 +493,10 @@ class ExecCGCommand : public Command {
 
   detail::CG &getCG() const { return *MCommandGroup; }
 
-  // MEmptyCmd one is only employed if this command refers to host-task.
-  // MEmptyCmd due to unreliable mechanism of lookup for single EmptyCommand
-  // amongst users of host-task-representing command. This unreliability roots
-  // in cleanup process.
+  // MEmptyCmd is only employed if this command refers to host-task.
+  // Due to unreliable mechanism of lookup for single EmptyCommand amongst users
+  // of host-task-representing command. This unreliability roots in cleanup
```
Comment (dm-vodopyanov) [inline sycl/source/detail/scheduler/commands.hpp:498]: If I understand correctly, last two sentences of the comment should be merged.
```suggestion
  // of host-task-representing command, it requires the cleanup process.
```

Comment (DenisBakhvalov) [reply sycl/source/detail/scheduler/commands.hpp:498]: Maybe like this:
The mechanism of lookup for single EmptyCommand amongst users of host-task-representing command is unreliable. This unreliability roots in the cleanup process.

Comment (dm-vodopyanov) [reply sycl/source/detail/scheduler/commands.hpp:498]: Denis's suggestion is more clear for me. @s-kanaev are you ok with this? Can you change this?

Comment (s-kanaev) [reply sycl/source/detail/scheduler/commands.hpp]: Done

**Comment by s-kanaev** [comment]
@dm-vodopyanov , @DenisBakhvalov ping

**Review by DenisBakhvalov — APPROVED** [review]

**Comment by s-kanaev** [comment]
@bader, please, merge if no objections

**Pull request merged**